src/Listener/PostChatGPTAnswer.php

```diff
@@ -40,6 +40,19 @@ public function handle(Started $event): void
         }
 
         $discussion = $event->discussion;
+
+
+        /** get discussion's all tag,if one of them is is_chatgpt,go on , or return*/
+        $discussionTags = $discussion->tags;
+        $ischatgpt = false;
+        foreach ($discussionTags as $discussionTag) {
+            if ((bool) $discussionTag->is_chatgpt) {
+                $ischatgpt = true;
+                break;
+            }
+        }
+        if (!$ischatgpt) return;
```

Comment:
```suggestion
        if (! $ischatgpt) {
            return;
        }
```

I prefer to use full syntax here

Reply:
;) It's okay.